pinc/MARCRecord.inc

```diff
@@ -98,24 +354,30 @@ class MARCRecord
     {
         $arthor = "";
```

Comment:
`$author`?

Reply:
Yeah, this needs to match the other updates.

Reply:
Woops. It most certainly does!